Ports/yasm/package.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=yasm
+version=1.3.0
+files="https://www.tortall.net/projects/yasm/releases/yasm-${version}.tar.gz yasm-${version}.tar.gz"
```

Comment:
This should use some form of integrity verification. Signature is best, otherwise a hard coded-known-good shasum.

Reply:
I couldn't find a signature for it so I added it manually. I hope I did it right

> Tests failed as `yasm` wasn't added to [Ports/AvailablePorts.md](https://github.com/SerenityOS/serenity/blob/master/Ports/AvailablePorts.md):
> 
> ```
>  [OK]: Meta/lint-shell-scripts.sh
> Running Meta/lint-ports.py... 
> AvailablePorts.md is missing the following ports:
>     yasm
> [FAIL]: Meta/lint-ports.py
> ```

I have now added this

Reply:
I couldn't find a signature so I added it manually I hope I did it right.